### DISCUSSION TITLE
Climate Page not showing

### DISCUSSION THREAD

**Discussion by malgates** [Blackymas/NSPanel_HA_Blueprint#2070] [Q&A]
Hi everyone,

Great project!

I am on Version 4.3.4 and I am struggeling to get the Climate Page working.

Here is my ESP HOME config:

`substitutions:
  # Settings - Editable values
  device_name: "nspanelwozi"
  friendly_name: "NS Panel Wohnzimmer"
  wifi_ssid: !secret wifi_ssid
  wifi_password: !secret wifi_password
  nextion_update_url: "http://homeassistant.local:8123/local/nspanel_eu.tft"  # Optional
  # Add-on configuration (if needed)
  heater_relay: "1"  # Possible values: "1" or "2"
  temp_min: "15" 
  temp_max: "23" 
  temp_step: "0.1"
  cold_tolerance: "0.1"
  hot_tolerance: "0.1"

climate:
  - id: !extend thermostat_embedded
    visual:
      temperature_step: ${temp_step}
    preset:
      - name: Home
        default_target_temperature_low: 19.0
        mode: "heat"
# Customization area
##### My customization - End #####

# Core and optional configurations
packages:
  remote_package:
    url: https://github.com/Blackymas/NSPanel_HA_Blueprint
    ref: main
    refresh: 300s
    files:
      - nspanel_esphome.yaml # Basic package
      # Optional advanced and add-on configurations
      # - esphome/nspanel_esphome_advanced.yaml
      # - nspanel_esphome_addon_climate_cool.yaml
      - nspanel_esphome_addon_climate_heat.yaml
      # - nspanel_esphome_addon_climate_dual.yaml`

In the Blueorint under the section "climate" and "Main climate Entity (Optional)" I added the entity "climate.nspanelwozi_thermostat". I set "Main Climate - Always Show Chip" to "on" and restarted the NSPanel. I will be greeted with the Termostat page, and cannot select anything. When I jump to the main screen I can no longer fo to the thermostat. 

I am a little puzzled right now. Is there not thermostat page? Am I forgetting something?

THX

**Comment by malgates** [comment]
Anyone using it as a thermostat?

**Reply by edwardtfn** [reply]
A lot of people, but usually this is reported as an issue, so you will find more cases like this in another area.
Anyways, I belive this was fixed with v4.3.6.
